### PR TITLE
RF | EntityFieldsOpsViewTest >testDeleteExistingRecord

### PR DESCRIPTION
**Pull request by AnastasiiaPV**
RF: https://trello.com/c/nLZFMmOi/860-rf-entityfieldsopsviewtest-testdeleteexistingrecord

### CODE REVIEW

File: src/test/java/EntityFieldsOpsViewTest.java [hunk 1]
```diff
@@ -132,6 +135,10 @@ private void sendValueToMultireferenceEmbedFO(String key){
         clickSendKeys(multireferenceField,key);
     }
 
+    private String getText(By by) {
```
Comment (Stan575) [inline src/test/java/EntityFieldsOpsViewTest.java:138]: would be nice to replace 'by' to something like 'locator' for better readability

File: src/test/java/EntityFieldsOpsViewTest.java [hunk 2]
```diff
@@ -270,6 +277,20 @@ public void testEditExistingRecord(){
 
     @Test
     public void testDeleteExistingRecord() {
+        testCreateRecord();
+
+        Assert.assertEquals(getDriver().findElements(By.xpath("//tbody/tr")).size(),1);
+        Assert.assertFalse(getText(CARD_BODY).isEmpty());
```
Comment (Stan575) [inline src/test/java/EntityFieldsOpsViewTest.java:283]: when you calling testCreateRecord(), that test does multiple assertions, lines 282 and 283 are not necessary in this test

**Review by sviet-ka — APPROVED** [review]

File: src/test/java/EntityFieldsOpsViewTest.java [hunk 3]
```diff
@@ -287,18 +308,15 @@ public void testDeleteExistingRecord() {
...
-        Assert.assertNotNull(getDriver().findElement(By.xpath(recycleBinExclamationPoint)));
+        testDeleteExistingRecord();
 
         findAndClick(By.xpath(recycleBinExclamationPoint));
 
         Assert.assertTrue(getDriver().findElement(By.xpath("//b[text()='Recycle Bin']")).isDisplayed());
         Assert.assertEquals(getListOfElements(By.xpath("//tbody//tr")).size(),1);
+        Assert.assertEquals(getDriver()
+                .findElement(By.xpath("//span[contains(text(), 'Showing 1 to 1 of 1 rows')]"))
+                .getText(),"Showing 1 to 1 of 1 rows");
```
Comment (Stan575) [inline src/test/java/EntityFieldsOpsViewTest.java:319]: this is not good practice to assert text from an element found by that particular text, need to be changed
for instance, locator can be:
`//span[@class='pagination-info']`

**Review by Stan575 — CHANGES_REQUESTED** [review]
overall great job!
please change locator in line 318

File: src/test/java/EntityFieldsOpsViewTest.java (hunk 3 above)
Comment (Stan575) [reply src/test/java/EntityFieldsOpsViewTest.java:319]: or use different locator type:
`By.className("pagination-info")`

**Review by omalanova — APPROVED** [review]

File: src/test/java/EntityFieldsOpsViewTest.java (hunk 1 above)
Comment (AnastasiiaPV) [reply src/test/java/EntityFieldsOpsViewTest.java]: Thank you for all the advice. This one is corrected.

File: src/test/java/EntityFieldsOpsViewTest.java (hunk 2 above)
Comment (AnastasiiaPV) [reply src/test/java/EntityFieldsOpsViewTest.java]: Done

File: src/test/java/EntityFieldsOpsViewTest.java (hunk 3 above)
Comment (AnastasiiaPV) [reply src/test/java/EntityFieldsOpsViewTest.java]: Done, I used the first one that you recommended

**Review by Stan575 — APPROVED** [review]
Good job!

**Pull request merged**